dask/dataframe/tests/test_dataframe.py

```diff
@@ -1675,15 +1675,43 @@ def test_sorted_index_single_partition():
         df.set_index('x'))
 
 
-def test_info(capsys):
-    df = pd.DataFrame({'long_column_name': [1, 2, 3, 4], 'short_name': [1, 0, 1, 0]})
-    ddf = dd.from_pandas(df, npartitions=1)
-    ddf.info()
-    out, err = capsys.readouterr()
-    assert out == ("<class 'dask.dataframe.core.DataFrame'>\n"
-                  "Data columns (total 2 columns):\n"
-                  "long_column_name    int64\n"
-                  "short_name          int64\n")
+def test_info():
+    from io import StringIO
+    from dask.compatibility import unicode
+    from pandas.formats import format
+    format._put_lines = put_lines
```

Comment:
Also it looks like this fails under Pandas 0.18.0.  Probably not a big deal if this is just in a test.  We could probably skip the test for versions under 0.18.1